mobile/src/main/java/org/openhab/habdroid/ui/activity/WebViewFragment.kt

```diff
@@ -77,6 +101,31 @@ class WebViewFragment : Fragment(), ConnectionFactory.UpdateListener {
         }
     }
 
+    override fun onCreateOptionsMenu(menu: Menu?, inflater: MenuInflater?) {
+        shortcutInfo.let { inflater?.inflate(R.menu.webview_menu, menu) }
```

Comment:
I wonder whether we only should do this pre-N? In N, one can use the launcher icon to pin the shortcut.

Reply:
Yes, users can pin the shortcut, but
1. it's not so visible to the user
2. maybe not every launcher supports this
3. if we add more WebViewFragments, we cannot add a shortcut for them, because only 4 to 5 are supported.